cloudify_cli/__init__.py

```diff
@@ -13,3 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ############
+
+import sys
+PY2 = sys.version_info[0] == 2
```

Comment:
you can also import this from `cloudify._compat`. Using that c-common one is the reason c-cli never grew a `_compat.py` itself